src/saucelabs/job.js

```diff
@@ -150,13 +142,29 @@ export default class Job {
 
         this.status = Job.STATUSES.INIT_BROWSER;
 
+        var executeCommand = WebDriverProtocol['/session/:sessionId/execute/sync'];
+        
+        WebDriverProtocol[`/session/:sessionId/execute`] = executeCommand;
+
         try {
-            await this.browser.init(initBrowserParams);
+            this.browser = await WebDriver.newSession({
+                protocol:               'http',
+                hostname:               'ondemand.saucelabs.com',
```

Comment:
Create a constant variable SAUCE_API_HOST for saucelabs.com and replace all appearance with it